main.js

```diff
@@ -99,8 +96,25 @@ const jsonButton = document.getElementById('jsonButton');
 const addButton = document.getElementById('addButton');
 const resizeSelectedButton = document.getElementById('resizeSelectedButton');
 
+var activelyDragging = false; 
+
+function getSelectedElementId() {
+    return selectedElement.parentElement.getAttribute("id").substr(-1); 
+}
+
+const mouseUp = document.addEventListener('mouseup', function(e){
+    const target = e.target
+    if (target.classList.contains("widget-box")) {
+        if (e.type === "mouseup") {
+            selectCell(target);  
+        }
+    } else if (target.nodeName != "BUTTON") {
```

Comment:
* `!==`

Always use `===` and `!==` when making comparisons in JS. `==` and `!=` will try to coerce objects, which can result in unexpected behavior.

Reply:
Thanks! I did not know that